src/ostorlab/cli/scan/run/assets/ip.py

```diff
@@ -16,22 +16,24 @@
 
 
 @run.run.command(name='ip')
-@click.argument('ip', required=True)
+@click.argument('ips', required=True, nargs=-1)
 @click.pass_context
-def ip_cli(ctx: click.core.Context, ip: str) -> None:
+def ip_cli(ctx: click.core.Context, ips: str) -> None:
     """Run scan for IP asset."""
     runtime = ctx.obj['runtime']
     try:
-        ip_network = ipaddress.ip_network(ip, strict=False)
-        if ip_network.version == 4:
-            asset = ipv4.IPv4(host=ip_network.network_address.exploded, mask=str(ip_network.prefixlen))
-        elif ip_network.version == 6:
-            asset = ipv6.IPv6(host=ip_network.network_address.exploded, mask=str(ip_network.prefixlen))
-        else:
-            raise NotImplementedError()
+        assets = []
+        for ip in ips:
+            ip_network = ipaddress.ip_network(ip, strict=False)
+            if ip_network.version == 4:
+                assets.append(ipv4.IPv4(host=ip_network.network_address.exploded, mask=str(ip_network.prefixlen)))
+            elif ip_network.version == 6:
+                assets.append(ipv6.IPv6(host=ip_network.network_address.exploded, mask=str(ip_network.prefixlen)))
+            else:
+                raise NotImplementedError()
```

Comment:
Instead of this exception, can you replace it with a console error and pass on the value.